### PR TITLE
#3042 C_Bpartner: new Additional fields 

### PR DESCRIPTION
**Pull request by cristinamghita**
#3042 C_Bpartner: new Additional fields

### CODE REVIEW

File: de.metas.adempiere.adempiere/base/de.metas.adempiere.adempiere.base_mf15-GenerateModel-table.launch [hunk 1]
```diff
@@ -8,10 +8,16 @@
...
+<booleanAttribute key="org.eclipse.jdt.launching.DEFAULT_CLASSPATH" value="true"/>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="org.adempiere.util.GenerateModel"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="${project_loc:de.metas.adempiere.adempiere.base}/src/main/java-gen&#13;&#10;&quot;-&quot;&#13;&#10;&quot;'%'&quot;&#13;&#10;&quot; '${string_prompt:TableName}' &quot;"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="de.metas.adempiere.adempiere.base"/>
 <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-DPropertyFile=&quot;${project_loc:metasfresh-dist}/metasfresh.properties&quot;&#13;&#10;-Dorg.adempiere.util.GenerateModel.OnlySystemColumns=true&#13;&#10;-Dgenerate.legacy=false"/>
+<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="${jrebel_args}&#13;&#10;-DPropertyFile=&quot;${project_loc:metasfresh-dist}/metasfresh.properties&quot;&#13;&#10;-Dorg.adempiere.util.GenerateModel.OnlySystemColumns=true&#13;&#10;-Dgenerate.legacy=false"/>
```
Comment (teosarca) [inline de.metas.adempiere.adempiere/base/de.metas.adempiere.adempiere.base_mf15-GenerateModel-table.launch]: i think this is not OK. not everybody is using jrebel.
more, what was the reason of chaging this file?

File: de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/ModelValidationEngine.java [hunk 2]
```diff
@@ -1165,16 +1179,11 @@ private void fireDocValidate(final PO po, final int docTiming, final List<ModelV
...
 
-	/**************************************************************************
-	 * Add Date Import Validation Listener
-	 *
-	 * @param tableName table name
-	 * @param listener listener
-	 */
-	public void addImportValidate(String importTableName, IImportValidator listener)
+	@Override
+	public void addImportValidate(String importTableName, IImportInterceptor listener)
```
Comment (teosarca) [inline de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/ModelValidationEngine.java]: rename it to addImportInterceptor

File: de.metas.adempiere.adempiere/base/de.metas.adempiere.adempiere.base_mf15-GenerateModel-table.launch (hunk 1 above)
Comment (cristinamghita) [reply de.metas.adempiere.adempiere/base/de.metas.adempiere.adempiere.base_mf15-GenerateModel-table.launch]: By mistake, I guess. The change wasn't intentional.

File: de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/ModelValidationEngine.java (hunk 2 above)
Comment (cristinamghita) [reply de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/ModelValidationEngine.java]: Ah.. right!

**Pull request merged**